org.eclipse.acute/src/org/eclipse/acute/AcutePlugin.java

```diff
@@ -57,4 +58,11 @@ public static void logError(Throwable t) {
 		getDefault().getLog().log(new Status(IStatus.ERROR, PLUGIN_ID, t.getMessage(), t));
 	}
 
+	public static String getDotnetCommand() {
+		IPreferenceStore store = plugin.getPreferenceStore();
+		if (store.getBoolean(AcutePreferenceInitializer.useExplicitDotnetPathPreference)) {
```

Comment:
I don't think we need that check. Isn't it simpler/better to always use the preference?